### PR TITLE
docs(rxDiskSize): FRMW-172-Migrate rxDiskSize to API docs

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-172

- [x] Dev LGTM
- [x] Dev LGTM

Migrate rxDiskSize to API documentation.

### CODE REVIEW

**Comment by bry4n** [comment]
LGTM

File: src/rxDiskSize/rxDiskSize.js [hunk 1]
```diff
@@ -13,8 +13,17 @@ angular.module('encore.ui.rxDiskSize', [])
...
  * @ngdoc filter
  * @name rxDiskSize.filter:rxDiskSize
- * @description [TBD]
- */
+ * @description
+ *
+ * Converts GB disk size into a more readable format (e.g. GBs, TBs, PBs)
+ *
+ * 
+ * ```
```
Comment (CITguy) [inline src/rxDiskSize/rxDiskSize.js]: For consistency, please use a `<pre>...</pre>` block.

**Comment by 100stacks** [comment]
:+1:

**Comment by CITguy** [comment]
LGTM

**Pull request merged**